spinalcordtoolbox/deepseg/models.py

```diff
@@ -136,7 +144,10 @@
          'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']},
     'seg_gm_sc_7t_t2star':
         {'description': 'SC/GM seg on T2*-weighted at 7T.',
-         'models': ['model_7t_multiclass_gm_sc_unet2d']}
+         'models': ['model_7t_multiclass_gm_sc_unet2d']},
+    'seg_lumbar_sc_t2w':
+        {'description': 'Lumbar SC segmentation on T2w contrast with 3D UNet.',
+        'models': ['model_seg_epfl_t2w_lumbar_sc']}
```

Comment:
This change will help to fix one of the errors from the [failed linting check](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/6011424804?check_suite_focus=true):

```suggestion
        {'description': 'Lumbar SC segmentation on T2w contrast with 3D UNet.',
         'models': ['model_seg_epfl_t2w_lumbar_sc']}
```